bindings/python/python/opendal/__init__.pyi

```diff
@@ -73,6 +73,8 @@ class AsyncOperator:
     async def copy(self, source: str, target: str): ...
     async def rename(self, source: str, target: str): ...
     async def remove_all(self, path: str): ...
+    @property
+    def operator(self) -> Operator: ...
```

Comment:
How about adding the following API to make this API more clear on what it does?

- `to_async_operator()` for `Operator` class
- `to_operator` for `AsyncOperator` class

Reply:
This could be better

```python
class Operator:
    @classmethod
    def from_config(scheme: str, **kwargs)-> Operator: ...

    @classmethod
    def from_async_operator(operator: AsyncOperator) -> Operator: ...

class AsyncOperator:
    @classmethod
    def from_config(scheme: str, **kwargs)-> AsyncOperator: ...

    @classmethod
    def from_operator(operator: Operator) -> AsyncOperator: ...
```

We use the  factory mode instead of `__init__` completely

Reply:
Seems cool, we can split as a new issue to start discussion.